src/private/index.ts

```diff
@@ -1,4 +1,5 @@
 export { menus } from './menus';
+export { files } from './files';
```

Comment:
Rename this to match whatever you decide to call the namespace (i.e. `log` or `logs`).